src/containers/WishListsPage.tsx

```diff
@@ -22,11 +22,10 @@
 import React from 'react';
 import intl from 'react-intl-universal';
 import { RouteComponentProps } from 'react-router-dom';
-import { WishListMain } from '../components/src/index';
 import { login } from '../utils/AuthService';
 import { cortexFetch } from '../utils/Cortex';
 import Config from '../ep.config.json';
-
+import WishListMain from '../components/src/WishList/wishlist.main';
```

Comment:
Let's keep the json import after the components.

Reply:
Sure thing ill make that change